scripts/lib/node_cache.py

```diff
@@ -62,40 +63,30 @@ def setup_node_modules(production=DEFAULT_PRODUCTION, stdout=None, stderr=None,
                         yarn_args,
                         success_stamp,
                         stdout=stdout,
-                        stderr=stderr,
-                        copy_modules=copy_modules)
+                        stderr=stderr)
 
     print("Using cached node modules from %s" % (cached_node_modules,))
-    cmds = [
-        ['rm', '-rf', 'node_modules'],
-        ["ln", "-nsf", cached_node_modules, 'node_modules'],
-    ]
-    for cmd in cmds:
-        run(cmd, stdout=stdout, stderr=stderr)
+    if os.path.islink('node_modules'):
+        os.remove('node_modules')
+    elif os.path.isdir('node_modules'):
+        shutil.rmtree('node_modules')
+    os.symlink(cached_node_modules, 'node_modules')
```

Comment:
So, one thing that this change is the logging on what `node_modules` symlink was created.

Reply:
The `print("Using cached node modules from %s" % (cached_node_modules,))` already provides that information.